src/BuiltinExtensions/ComfyUIBackend/ExtraNodes/SwarmComfyCommon/SwarmSaveImageWS.py

```diff
@@ -12,7 +13,8 @@ class SwarmSaveImageWS:
     def INPUT_TYPES(s):
         return {
             "required": {
-                "images": ("IMAGE", )
+                "images": ("IMAGE", ),
+                "bit_depth": (["8", "16"], {"default": "8"})
```

Comment:
needs to be under `optional` as its a new input (otherwise it's a breaking change on presaved workflows)